grapher/rule.go

```diff
@@ -17,6 +18,7 @@ import (
 )
 
 const graphOp = "graph"
+const goUnitType = "GoPackage"
```

Comment:
Is there any way we can avoid special casing here? Ideally, srclib core should not contain any language-specific code. It's very easy for this to become a tangled mess of special cases / if statements.